sdk/monitor/azure-monitor-opentelemetry/samples/tracing/django/sample/manage.py

```diff
@@ -15,7 +15,9 @@ def main():
     # Configure Azure monitor collection telemetry pipeline
     # configure_azure_monitor should only be called once in either asgi.py, wsgi.py, or manage.py, depending on startup method.
     # If using manage.py, please remove configure_azure_monitor from asgi.py and wsgi.py
-    configure_azure_monitor()
+    configure_azure_monitor(
+        logger_name="app_logger",
```

Comment:
I think if we are not using the root logger, we should at least use a valid module to show Cxs how to instrument a logging at the module level for their app.